extensions/editor.js

```diff
@@ -6,8 +6,8 @@ import './shared/block-category';
 import './shared/plan-upgrade-notification';
 import './shared/stripe-connection-notification';
 import './shared/external-media';
-import './extended-blocks/paid-blocks';
 import './extended-blocks/core-embed';
+import './extended-blocks/paid-blocks';
```

Comment:
minor suggestion: this change is not neccessary.

Reply:
Thanks for pointing it out! This happened when I was rebasing the PR. I'm thinking of keeping it, since it alphabetizes imports from the `./extended-blocks` folder neatly :sweat_smile: